tests/rptest/tests/mirror_maker_test.py

```diff
@@ -146,57 +147,73 @@ def test_simple_end_to_end(self, source_type):
             self.logger.debug(f'source topic: {t}, target topic: {desc}')
             assert len(desc.partitions) == t.partition_count
 
-    @cluster(num_nodes=9)
+    @cluster(num_nodes=10)
     @parametrize(source_type=kafka_source)
-    @parametrize(source_type=redpanda_source)
+    #@parametrize(source_type=redpanda_source)
```

Comment:
Accidental commit of commenting this line out?

Reply:
yeah, i did to make turnaround faster, fixing